tests/run.sh

```diff
@@ -13,26 +13,26 @@ set -eu
 scrutinizer=${SCRUTINIZER:-false}
 
 # only use sudo if available e.g., alpine vs ubuntu
-hasSudo='sudo';
+sudoCmd='sudo'
```

Comment:
also I would put default `''` and if it exists, use it, so invert the logic, as positive ifs are always better ;)